### ISSUE TITLE
Info Request.

### ISSUE THREAD

**Issue by Alfredoxrock** [ChiefNoir/ss-pbc#131]
Is this a personal project?
Can I help in any way?

**Comment by ChiefNoir** [comment]
Yep, it's a purely personal project of a personal website dedicated to _me_. 

I'm building it for fun, for showcasing, and for flexing around. 
I don't think that someone will go throw a complicated process of deployment (also, the instruction is not ready yet) to get a two-pages-personal-web-site purely built to my tastes.

Well, if you want to, you can always go throw the code, looking for bad code, bad practices, generally bad things, or vulnerabilities.
I don't think, I will implement or need feature requests, because, well, it is my site and right now I don't need new features.
